tests/entrypoints/openai/test_optional_middleware.py

```diff
@@ -20,6 +20,12 @@
 
 @pytest.fixture(scope="module")
 def server(request: pytest.FixtureRequest):
+    # ROCm SPECIFIC CONFIGURATION:
```

Comment:
Isnt a better and more user friendly solution to this problem to make the default attn backend for this type of model be FLEX_ATTENTION?